### ISSUE TITLE
transfer learning on the base categories does not help detection on the novel categories

### ISSUE THREAD

**Issue by twangnh** [microsoft/RegionCLIP#4]
Hi, Thanks for sharing the work! I find that the open-vocabulary object detection result on novel categories is not much better than that of zero-shot one, I'm wondering if fine-tuning on the base categories would actually hurt the performance?

**Comment by twangnh** [comment]
I see, thanks for your reply. So for the zero-shot inference experiment, if you use the LVIS 866 categories trained RPN, would that be unfair since the 17 novel COCO categories are likely seen in the 866 categories?

**Comment by YiwuZhong** [comment]
Note that in zero-shot inference setting, there is no "novel" or "base" in fact, since no detection labels were used during pre-training. The reason we showed novel and base AP50 in Table 4 is for the consistency with previous tables.

In Table 4, all models used the same regions (e.g., regions from RPN trained by 866 categories, ground-truth regions). The only difference is the backbone initialization (e.g., CLIP, RegionCILP). So it is fair to compare the models **within** Table 4 for evaluating the region recognition performance.
